src/harness/vfs.ts

```diff
@@ -404,7 +409,21 @@ namespace vfs {
         }
 
         /**
-         * Get file status.
+         * Change file access times
+         *
+         * NOTE: do not rename this method as it is intended to align with the same named export of the "fs" module.
+         */
+        public utimesSync(path: string, atime: Date, mtime: Date) {

```

Comment:
I originally had this as part of the VFS, the relevant code can be found here: https://gist.github.com/rbuckton/1d064ec108afc0479bc1457d8a46792e#file-filesystem-ts-L596

You should verify the file system isn't read-only before performing updates so that you don't inadvertently modify a copy of the FS shared between multiple tests, i.e.:

```ts
if (this.isReadOnly) throw new IOError("EROFS");
```